Objects/file formats/XGAssemblyCodeExtensions.swift

```diff
@@ -132,6 +132,22 @@ extension XGAssembly {
 	}
 	#endif
 
+	class func replaceRamASM(RAMOffset: Int, newASM asm: [UInt32]) {
```

Comment:
Do you want to keep this? I think I removed it and switched to the metroworks area in dol as the preference. This could still be useful in future if we end up writing a lot of assembly code. It's region specific though which is the main reason I didn't like it as a solution.

Reply:
This is actually just the `UInt32` version of the function, they are identical with different return types